### ISSUE TITLE
AccessibilityLabel fail when Text is constantly updated

### ISSUE THREAD

**Issue by lucasaugustin** [facebook/react-native#31371]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
When a Text component has its content updated constantly (like in a countdown), the Android talkBack starts to read Text AccessibilityLabel and the Text content.

## React Native version:
System:
    OS: macOS 10.15.7
    CPU: (8) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
    Memory: 45.17 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 13.8.0 - /usr/local/bin/node
    Yarn: 1.22.0 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.2, 29.0.3
      System Images: android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 4.0 AI-193.6911.18.40.6626763
    Xcode: 12.4/12D4e - /usr/bin/xcodebuild
  npmPackages:
    react: 16.12.0 => 16.12.0 
    react-native: 0.61.5 => 0.61.5

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Create a Text component that gets constantly updated
2. Use Android TalkBack to read the component

## Expected Results
Its expected that the Android TalkBack reads only the accessibilityLabel.

## Snack, code example, screenshot, or link to a repository:
this is all that is necessary to reproduce the bug:

`  const [time, setTime] = useState(60)
  useEffect(() => {
    if (time > 0) { setTimeout(() => { setTime(time - 1) }, 3000) }
  }, [time])
  return (
    <View>
      <Text accessibilityLabel="accessible Text">
        {`remaining time: ${time}`}
      </Text>
    </View>
  )`

**Comment by fabriziobertoglio1987** [comment]
I'm going to try to reproduce this. Thanks
